python_modules/automation/automation/scaffold/assets/dagster-example-tmpl/tox.ini.tmpl

```diff
@@ -1,13 +1,14 @@
 [tox]
-envlist = py{39,38,37,36}-{unix,windows},mypy,pylint
+envlist = py{39,38,37},mypy,pylint
 skipsdist = True
 
 [testenv]
 setenv =
-  VIRTUALENV_PIP=21.3.1
+  VIRTUALENV_PIP=22.1.2
 passenv = CI_* COVERALLS_REPO_TOKEN BUILDKITE
 deps =
   -e ../../python_modules/dagster[mypy,test]
+  -e ../../python_modules/dagit
```

Comment:
why do we need dagit as a test dep?

Reply:
Can't remember my reasoning but now it's a moot point if we're deleting this code.